wrapper.go

```diff
@@ -8,17 +8,18 @@ import (
 
 // These values define known types of Wrappers
 const (
-	AEAD           = "aead"
-	Shamir         = "shamir"
-	PKCS11         = "pkcs11"
-	AliCloudKMS    = "alicloudkms"
-	AWSKMS         = "awskms"
-	GCPCKMS        = "gcpckms"
-	AzureKeyVault  = "azurekeyvault"
-	OCIKMS         = "ocikms"
-	HuaweiCloudKMS = "huaweicloudkms"
-	Transit        = "transit"
-	Test           = "test-auto"
+	AEAD            = "aead"
+	Shamir          = "shamir"
+	PKCS11          = "pkcs11"
+	AliCloudKMS     = "alicloudkms"
+	AWSKMS          = "awskms"
+	GCPCKMS         = "gcpckms"
+	AzureKeyVault   = "azurekeyvault"
+	OCIKMS          = "ocikms"
+  HuaweiCloudKMS = "huaweicloudkms"
```

Comment:
Formatting changed for some reason

Reply:
Formatting changed due to github conflict merge, fixed.
The tests of HuaweiCloudKMS never passed, I also fixed it.